cluster.yml

```diff
@@ -1,11 +1,7 @@
 ---
-- hosts: downloader
```

Comment:
it make sense to keep. 
k8s-cluster means  you want to run roles on all nodes.
downloader can be any machine that can access the cluster.